support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -193,18 +197,68 @@ function Heading({
 	amount,
 	currency,
 	contributionType,
+	ratePlanKey,
 	paymentStatus,
 	promotion,
 }: HeadingProps): JSX.Element {
 	const isPending = paymentStatus === 'pending';
 	const isDigitalEdition = productKey === 'DigitalSubscription';
 	const isGuardianAdLite = productKey === 'GuardianAdLite';
 	const isTier3 = productKey === 'TierThree';
+	const paperProductKeys: ActiveProductKey[] = [
+		'NationalDelivery',
+		'HomeDelivery',
+		'SubscriptionCard',
+	];
+	const guardianWeeklyProductKeys: ActiveProductKey[] = [
+		'GuardianWeeklyDomestic',
+		'GuardianWeeklyRestOfWorld',
+	];
+	const isPaperProduct = paperProductKeys.includes(productKey);
+	const isPrintProduct =
+		isPaperProduct || guardianWeeklyProductKeys.includes(productKey);
```

Comment:
Since we are not using those variables elsewhere, it might be simpler to evaluate all in one place.
```suggestion
	const printProducts: ActiveProductKey[] = [
		'NationalDelivery',
		'HomeDelivery',
		'SubscriptionCard',
		'GuardianWeeklyDomestic',
		'GuardianWeeklyRestOfWorld',
	];

	const isPrintProduct = printProducts.includes(productKey);
```